meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -53,9 +53,11 @@ do_install_append_lmp() {
     install -m 0644 ${WORKDIR}/tmpfiles.conf ${D}${nonarch_libdir}/tmpfiles.d/aktualizr-lite.conf
 }
 
-PACKAGES += "${PN}-get ${PN}-lite"
+PACKAGES += "${PN}-get ${PN}-lite ${PN}-lite-lib"
 FILES_${PN}-get = "${bindir}/aktualizr-get"
 FILES_${PN}-lite = "${bindir}/aktualizr-lite"
+FILES_${PN}-lite-lib = "${libdir}/libaktualizr_lite.so"
+FILES_${PN}-dev += "${includedir}/${PN}-lite"
```

Comment:
once https://github.com/foundriesio/aktualizr-lite/pull/121 is merged, this will need to be:
```
FILES_${PN}-dev += "${includedir}/${PN}-lite ${includedir}/json"
```
If you want I can split this patch out and try and get it merged independently of this.

Reply:
> If you want I can split this patch out and try and get it merged independently of this.

Please, do it. I still need to put the finishing touches to the "restorable apps" and it makes sense to add another commit to the given PR that bumps the final version of it (aklite), so this PR is not ready to be merged, while the "libapi" related changes are need now.

Reply:
https://github.com/foundriesio/meta-lmp/pull/430